cmd/create/admin/cmd.go

```diff
@@ -83,8 +84,8 @@ func run(cmd *cobra.Command, _ []string) {
 	var err error
 	passwordArg := args.passwordArg
 	if len(passwordArg) == 0 {
-		r.Reporter.Debugf("Generating random password")
-		password, err = generateRandomPassword(23)
+		r.Reporter.Debugf(GeneratingRandomPasswordString)
+		password, err = GenerateRandomPassword(23)
```

Comment:
NP
Can we use a const here for the password length?

Reply:
I will go ahead and remove the param requirement of the length of the password, this method is solely used for admin user password generation. I will store the length of the password as a const where the method is defined.
/cmd/create/admin/cmd.go